manage-data/lifecycle/index-lifecycle-management/rollover.md

```diff
@@ -9,50 +9,96 @@ products:
+* **Optimized performance** - keeps shard sizes within recommended limits (10-50 GB)
+* **Queries run faster** - improves search performance
 
-* A name or wildcard (`*`) pattern for the data stream.
-* The data stream’s timestamp field. This field must be mapped as a [`date`](elasticsearch://reference/elasticsearch/mapping-reference/date.md) or [`date_nanos`](elasticsearch://reference/elasticsearch/mapping-reference/date_nanos.md) field data type and must be included in every document indexed to the data stream.
-* The mappings and settings applied to each backing index when it’s created.
+Rollover can be triggered via the [API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-rollover), ILM, or DLM.
 
-Data streams are designed for append-only data, where the data stream name can be used as the operations (read, write, rollover, shrink etc.) target. If your use case requires data to be updated in place, you can instead manage your time series data using [index aliases](../../data-store/aliases.md). However, there are a few more configuration steps and concepts:
+## How rollover works in ILM
 
-* An *index template* that specifies the settings for each new index in the series. You optimize this configuration for ingestion, typically using as many shards as you have hot nodes.
-* An *index alias* that references the entire set of indices.
-* A single index designated as the *write index*. This is the active index that handles all write requests. On each rollover, the new index becomes the write index.
+You define a rollover action in the hot phase of an index lifecycle policy. It will run when any of the configured thresholds are met and the write index contains at least one document.
+You can configure the following rollover conditions:
 
-::::{note}
-When an index is rolled over, the previous index’s age is updated to reflect the rollover time. This date, rather than the index’s `creation_date`, is used in {{ilm}} `min_age` phase calculations. [Learn more](../../../troubleshoot/elasticsearch/index-lifecycle-management-errors.md#min-age-calculation).
+* **Size** - an index will rollover when its shards reach a set size, for example 50 GB.
+* **Age** - an index will rollover when an index reaches a certain age, for example 7 days.
+* **Document count** - an index will rollover when a shard contains a certain number of documents, for example 2 million.
 
+::::{tip}
+Rolling over to a new index based on size, document count, or age is preferable to time-based rollovers. Rolling over at an arbitrary time often results in many small indices, which can have a negative impact on performance and resource usage.
 ::::
 
+After rollover, indices move to other index lifecycle phases like warm, cold, frozen, and delete. Rollover creates a new write index while the old one continues through the lifecycle phases.
```

Comment:
```suggestion
After rollover, indices move through other configured index lifecycle phases: warm, cold, frozen, and/or delete. Rollover creates a new write index while the old one continues through the lifecycle phases.
```
Would this be better? I'd probably avoid `like` since that seems to me to imply that "warm", "cold", etc. are examples, rather than an exhaustive list.

Reply:
Cool, that works! 
Why is there an `or` at the end of the list? Are the phases mutually exclusive?

Reply:
Right, thank you! 🙏